packages/web/hooks/one-click-trading/use-one-click-trading-session.ts

```diff
@@ -35,7 +35,10 @@ export const useOneClickTradingSession = ({
     };
 
     if (!account?.address) {
-      return defaultReturn;
+      // Return a promise that never resolves so loading is kept true
+      // until we can decide what to share
+      // without address 1CT session should not exist
+      return new Promise<typeof defaultReturn>(() => {});
     }
```

Comment:
It doesn’t seem logical to return isLoading as true when we know for certain that the session is `undefined`. Is this intended for a specific view? If so, we can simply set `isLoading` to true in that context when the session is `undefined`.